tests/test_tools.py

```diff
@@ -3,16 +3,20 @@
 
 from sympy.abc import a, b, c, d, e
 
-from devito.tools import partial_order
+from devito.tools import toposort
 
 
 @skipif_yask
 @pytest.mark.parametrize('elements, expected', [
     ([[a, b, c], [c, d, e]], [a, b, c, d, e]),
     ([[e, d, c], [c, b, a]], [e, d, c, b, a]),
-    ([[a, b, c], [b, d, e]], [a, b, c, d, e]),
+    ([[a, b, c], [b, d, e]], [a, b, d, c, e]),
```

Comment:
Note: both orderings would be fine, but this version of ``toposort`` returns ``d, c``, rather than ``c, d``. It is deterministic, and that's the only thing that really matters at this point